libibverbs/examples/devinfo.c

```diff
@@ -608,7 +608,7 @@ static int print_hca_cap(struct ibv_device *ib_dev, uint8_t ib_port)
 			      device_attr.max_dm_size);
 	}
 
-	for (port = 1; port <= device_attr.orig_attr.phys_port_cnt; ++port) {
+	for (port = 0; port++ < device_attr.orig_attr.phys_port_cnt;) {
```

Comment:
Don't do gross stuff like this, just make the port unsigned int.

Reply:
OK, the PR was updated accordingly.